src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -274,6 +274,18 @@
     type: command
     short-summary: Unregister a provider.
 """
+helps['provider operation'] = """
+    type: group
+    short-summary: get provider operations.
+"""
+helps['provider operation show'] = """
+    type: command
+    short-summary: Get an individual provider's operations.
+"""
+helps['provider operation list'] = """
+    type: command
+    short-summary: Get operations from all providers.
```

Comment:
It seems this doesn't need to be a group.  You could just have a "list-operations" under provider that optionally takes a provider as a parameter and then perform list/show as needed, just like with our "list/list-all" list commands.